src/app/cases/page.tsx

```diff
@@ -36,16 +32,14 @@ export default function Page() {
               key={c.id}
               caseData={c}
               isSelected={c.id === selectedCard}
-              onClick={() => setSelectedCard(c.id)}
+              onClick={() => {
+                setSelectedCard(c.id);
+                setCaseInfo(c);
+              }}
             />
           ))}
         </CardColumn>
-        <CaseDetailDisplay>
-          {/* proof of concept -- to turn into component later */}
-          <CaseDetails>
-            <H2>Case details.</H2>
-          </CaseDetails>
-        </CaseDetailDisplay>
+        {caseInfo && <CaseDetailDisplay caseData={caseInfo} />}
```

Comment:
is there ever a case where the caseInfo is False, and the CaseDetailDisplay doesn't show up?

Reply:
I believe so, when the user hasn't selected a card. Not sure if we want a selected card by default though.

Reply:
I think there is the case that initially `caseInfo` is empty, so there will be no case detail to display. Not sure what the consensus is on auto-selecting the first case though, but if we do that then there's the edge case of us having no cases to display.

Reply:
yup what jinkang said is correct, sorry for not replying to this earlier. When the page first renders, `caseInfo` is undefined as it's set whenever a case card is clicked. This causes an error when trying to pass it in as a prop to the `CaseDetailDisplay` component which is why we need to check that `caseInfo` is not null/undefined. Also with this current implementation, it looks like it defaults to displaying the first case already when the page first renders. I think maybe we should test this page for the edge case where no cases are currently on the backend, and decide on what we would want the behavior to look like?

Reply:
i'm a fan of auto-selecting the first case! i think we probably want to discuss this + what the interest form will look like if there are no cases. adding to tmr's meeting agenda, and also cc'ing @davidqing6432 on this thread